### DISCUSSION TITLE
Picture and Markdown

### DISCUSSION THREAD

**Discussion by PeterGyger** [community/community#22719] [New to GitHub]
<p>Hello</p>
<p>Again…</p>
<p>As often in recent years, images are suddenly no longer displayed in the browser. But on GitHub, they do.</p>
<p>For 48 hours, on multiple computers, different browsers reproducible</p>
<p>Markdown:<br>
<code>![bla](../_image/21-1.png)</code><br>
<code>![Win-DLNA Server](../_image/dlna-1.png)</code></p>
<p>Last Time:<br>
<a href="https://github.community/t/image-path-when-the-file-isnt-in-the-root/10499/3">https://github.community/t/image-path-when-the-file-isnt-in-the-root/10499/3</a></p>
<p>Static HTML has never been so unpredictable</p>
<p>Any idea?</p>

**Comment by SethClydesdale** [comment]
<p>Hello <a class="mention" href="/PeterGyger">@PeterGyger</a>,</p>
<p>I believe the issue is that your images are located in a directory that begins with an underscore (_). Jekyll typically ignores these directories and their contents. So, I would recommend renaming the “_image” folder to just simply “image” or place them in the assets/images folder. This should hopefully resolve the issue.</p>

**Comment by PeterGyger** [comment]
<p>Hello</p>
<p>It hasn’t worked for 2 days<br>
It works on Github<br>
It does not work in the browsers</p>
<p>Conclusion: “Jeckyl does not like underscores”?</p>
<p>Testing is better than believing, so I’m trying</p>

**Comment by SethClydesdale** [comment]
<p>Pretty much. The only files/folders that should include underscores at the beginning would be those that are listed <a href="https://jekyllrb.com/docs/structure/" rel="nofollow noopener">here</a>. If the file or folder that begins with an underscore is not one of these, it’ll usually be ignored by Jekyll.</p>

**Comment by PeterGyger** [comment]
<p>Hello</p>
<p>Thanks for the URL<br>
At the bottom of the page is how to do it if you want to use directories with underscore</p>
<p>I changed the directory to “image” - directly in the root. Modified the code.<br>
Unchanged: In GitHub OK, in browsers NOK</p>
<p>I saved the image file in jpg format, adapted the code and again NOK</p>
<p>Any Idea?</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Looks like you may need to move up one directory when referencing the image. Try changing:</p>
<pre><code class="lang-markdown">![bla](../_image/21-1.png)
</code></pre>
<p>to:</p>
<pre><code class="lang-markdown">![bla](/../_image/21-1.png)
</code></pre>
<p>Because currently it appears it’s trying to pull the image from the /praxis/ directory which does not contain an image folder. When moving up one directory it appears to work.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c53f61de8cd4c9fa2076c7817e91ccd558875de3.png" data-download-href="/uploads/short-url/s8VRaKikE9LBSED7uBQEcjdtm8j.png?dl=1" title="image-src" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181072706-abdcd166-3db9-4dd1-bf71-6ba47e8f0ca7.png" alt="image-src" data-base62-sha1="s8VRaKikE9LBSED7uBQEcjdtm8j" width="690" height="256" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c53f61de8cd4c9fa2076c7817e91ccd558875de3_2_690x256.png, https://aws1.discourse-cdn.com/github/optimized/2X/c/c53f61de8cd4c9fa2076c7817e91ccd558875de3_2_1035x384.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/c53f61de8cd4c9fa2076c7817e91ccd558875de3_2_1380x512.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c53f61de8cd4c9fa2076c7817e91ccd558875de3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image-src</span><span class="informations">1407×523 91.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by PeterGyger** [comment]
<p>Thanks for your persistent help</p>
<p>Unfortunately, the syntax adjustment to “/…/image…” NOK</p>
<p>Let’s go back to “Start”<br>
I have been posting with pictures on Github for several years<br>
Since 48 hours it works - only in the browser! - no more.<br>
If the syntax was the cause, it would also be displayed incorrectly on GitHub.</p>
<p>With David Wells’ “How-to”, I have now switched to HTML-img tag. And it works</p>
<p><a href="https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d" rel="nofollow noopener">https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d</a></p>
<p>Like I said, every couple of months I get weird phenomena with pictures and markdown on GitHub. You should stick to static HTML as much as possible</p>
<p>Thanks for your support</p>

**Comment by 577928** [comment]
![Uploading 17476002025328788689028539200281.jpg…]()

**Comment by 577928** [comment]
<a href="http://www.stanleyfashions.com"target="_blank">Visit stanley</a>
<img src="image 🪣😚🤫🤭🤭😄😁😭"
alt="🪣😚🤫🤭🤭😄😁😭"
